src/processes/follow-topic-process.js

```diff
@@ -134,18 +134,30 @@ const followTopicProcess = async (job, done) => {
 
         await chat.create();
         const channelState = await chat.watch();
-
         if (channelState.messages.length === 0) {
           await chat.sendMessage({
             text: communityMessageFormat.message,
-            user_id: communityMessageFormat.user_id
+            user_id: communityMessageFormat.user_id,
+            is_auto_message: true
           });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code is sending an automatic message if the channel has no messages. However, there's no error handling for `chat.sendMessage()`. If it fails due to any reason (like network issues), the error will not be caught and handled properly.
```diff
+ try {
  await chat.sendMessage({
    text: communityMessageFormat.message,
    user_id: communityMessageFormat.user_id,
    is_auto_message: true
  });
+ } catch (error) {
+   console.error('::: Error on sendMessage :::', error);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->